openring/openring_template_in.html

```diff
@@ -36,11 +41,15 @@ <h4 class="title">
   padding: 0.5rem;
   background: #eee;
   min-width: 10rem;
+  padding: 0.5rem;
+  margin: 0 0.5rem;
```

Comment:
Let's make the background transparent and a thin board with our default green color. Something like:
```
border-width: 1px;
border-style: solid;
border-color:
```
In the responsive mode, make sure that you add a proper space.